backend/routes/auth/isLoggedInAdmin.js

```diff
@@ -0,0 +1,11 @@
+function isLoggedInAdmin(req, res, next) {
+  if (
+    process.env.NODE_ENV === 'test' ||
+    (req.isAuthenticated() && req.user.role === 'admin')
+  ) {
```

Comment:
I prefer doing something like this:
```js
const isTestEnv = process.env.NODE_ENV === 'test';
const isAuthenticated = req.isAuthenticated();
const hasAdminRole = req.user.role === 'admin';

if (isTestEnv || (isAuthenticated && hasAdminRole) {
...
}
```
It's a bit longer, but much more readable.

Reply:
you have a point there, though I think the isAuthenticated is a bit OTT.